client/package.json

```diff
@@ -17,9 +18,18 @@
 	},
 	"scripts": {
 		"start": "react-scripts start",
-		"build": "npm i && react-scripts build && rm -rf ../server/client-build/ && mv build ../server/client-build/",
+		"build": "cross-os cross-os-build",
```

Comment:
I think indentation is off (too large).

Reply:
@tmathern I see 2 tabs in VSC
<img width="170" alt="Screen Shot 2020-05-29 at 10 14 15 AM" src="https://user-images.githubusercontent.com/1933320/83281341-2f100700-a195-11ea-8750-ceca5f2143dd.png">